File: Skill II/Day3/Part2/66.py
class Solution:
    def plusOne(self, digits: List[int]) -> List[int]:
        dStringsList = [str(d) for d in digits]
        dString = "".join(dStringsList)
        digit = int(dString)
        outputDigit = digit + 1
        output = [int(i) for i in str(outputDigit)]
        return output

File: Skill II/Day3/Part2/66.java
class solution{
    public int[] plusOne(int[] digits) {
    int n = digits.length;

    for (int i=n-1; i>=0; i--) {
        if(digits[i] < 9) {
            digits[i]++;
            return digits;
        }
        
        digits[i] = 0;
    }
    
    int[] newNumber = new int [n+1];
    newNumber[0] = 1;
    
    return newNumber;
    }
}

// 0ms java solution from dicuss.   
// https://leetcode.com/problems/plus-one/discuss/24082/My-Simple-Java-Solution